website/index.js

```diff
@@ -0,0 +1,16 @@
+document.addEventListener('DOMContentLoaded', () => {
+    console.log('App loaded');
+
+    hljs.initHighlightingOnLoad();
+
+    var jscode = document.getElementById('js-code')
+    jscode.style.backgroundColor = '#f9fbfc'
+    jscode.style.fontSize = '14px'
+    jscode.style.fontFamily = 'Roboto Mono'
+
+    var bashcode = document.getElementById('bash-code')
+    bashcode.style.backgroundColor = '#f9fbfc'
+    bashcode.style.fontSize = '14px'
+    bashcode.style.lineHeight = '30px'
+    bashcode.style.fontFamily = 'Roboto Mono'
```

Comment:
This type of styling shouldn't be accomplished like this. HighlightJS has themes [support](https://highlightjs.org/static/demo/). You can actually just add a stylesheet and it will handle this styling.


I've done it in my branch, check [this](https://github.com/hweeks/dynamoose/blob/jsdoc-to-markdown/docs/stylesheets/default.css) out.